keps/sig-node/4381-dra-structured-parameters/README.md

```diff
@@ -2026,6 +1947,29 @@ type PodResourceClaimStatus struct {
 }
 ```
 
+##### ResourceQuota
+
+The core ResourceQuota mechanism checks at admission time whether creating a
+resource in the apiserver would exceed per-namespace limits. This mechanism
+gets extended to support limiting the number of ResourceClaims based on what
+gets requested.
+
+A resource entry in ResourceQuota with the special
+`<device-class-name>.deviceclass.resource.k8s.io/devices` limits the number of
+ResourceClaim objects in a namespace such that the number of devices requested
+through those objects with that class does not exceed the limit. When admins
+define classes so that they select specific device types, this quota mechanism
+can be used to limit access to those device types.
```

Comment:
```suggestion
A resource entry in ResourceQuota with the name
`<device-class-name>.deviceclass.resource.k8s.io/devices` limits the number of
ResourceClaim objects in a namespace such that the number of devices requested
through objects with that device class does not exceed the specified limit. When admins
define device classes so that they select specific device types, this quota mechanism
can be used to limit access to those device types.
```